---
title: "plot Jump-seq reads distribution"
output: pdf_document
---

### plus reads and minus sites
```{r,echo=FALSE}
library(ggplot2)
library(Rmisc)
#function to return a data.frame that could be used to draw plots
orgnize_table <-function(data){
  #data is a table in which each column represents under one condition the counts of 5' end of reads across -100 to +100 of hydroxymethylation sites
  data2 = as.data.frame(cbind(apply(data[,1:5],2,function(x){x/sum(x)}),position = data$position)) # normalize for each column of the first 5
  data2 = reshape(data2, varying = list(1:5),v.names = "count",idvar = "position",times = c("48ng","24ng","12ng","6ng","2.4ng"),direction = "long")
  data1 = reshape(data, varying = list(1:5),v.names = "count",idvar = "position",times = c("48ng","24ng","12ng","6ng","2.4ng"),direction = "long")
  data1$time = factor(data1$time, levels = c("48ng","24ng","12ng","6ng","2.4ng"))
  data2$time = factor(data2$time, levels = c("48ng","24ng","12ng","6ng","2.4ng"))
  list(original = data1, normalized = data2)
}

### a vector containing different conditions
conditions = c("48ng","24ng","12ng","6ng","2.4ng")
```

```{r,echo=FALSE}
#plus reads and minus sites
setwd("/media/yuwen/Elements/Tiao_Seq/analysis")
mc5_48_plus = read.delim("../data/Data/160620/He-Lu-6_5mC-jump-48ng-S6_L004_R1_001.adaptor_removed.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

mc5_24_plus = read.delim("../data/Data/160620/He-Lu-7_5mC-jump-24ng-S7_L004_R1_001.adaptor_removed.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)


mc5_12_plus = read.delim("../data/Data/160620/He-Lu-8_5mc-jump-12ng-S8_L004_R1_001.adaptor_removed.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)


mc5_6_plus = read.delim("../data/Data/160620/He-Lu-9_5mc-jump-6ng-S9_L004_R1_001.adaptor_removed.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

mc5_2.4_plus = read.delim("../data/Data/160620/He-Lu-10_5mc-jump-2.4ng-S10_L004_R1_001.adaptor_removed.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)


mc5_plus = cbind(mc5_48_plus,mc5_24_plus,mc5_12_plus,mc5_6_plus,mc5_2.4_plus)
colnames(mc5_plus) = c("48ng","24ng","12ng","6ng","2.4ng")
mc5_plus = data.frame(mc5_plus, position = seq(-100,100,1))
```

```{r,echo=FALSE}
knitr::opts_chunk$set(fig.width = 10,fig.height = 5)
options(warn=-1)
```



* 5mc plus strands minus hydroxymethyl sites, unnormalized

```{r,echo=FALSE,fig.width=5,fig.height=5}
test = orgnize_table(mc5_plus)
ggplot(aes(x = position, y = count, color = time), data = test$original)+geom_line()+scale_color_discrete(guide = guide_legend(title = "Condition"))+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black",hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"),legend.position = c(.9, .8))+xlab("Distance")+ylab("Count")
```

```{r,echo=FALSE}
plots <- list()  # new empty list
for (i in 1:5) {
    p1 = ggplot(aes(x = position, y = count), data = test$original[test$original$time == conditions[i],])+geom_line()+ggtitle(conditions[i])+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))
    plots[[i]] <- p1  # add each plot into plot list
}
multiplot(plotlist = plots, cols = 2)
```

\newpage
* 5mc plus strands minus hydroxymethyl sites, normalized

```{r,echo=FALSE,fig.width=5,fig.height=5}
ggplot(aes(x = position, y = count, color = time), data = test$normalized)+geom_line()+scale_color_discrete(guide = guide_legend(title = "Condition"))+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black",hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"),legend.position = c(.9, .8))+xlab("Distance")+ylab("Frequency")
```

```{r,echo=FALSE}
plots <- list()  # new empty list
for (i in 1:5) {
    p1 = ggplot(aes(x = position, y = count), data = test$normalized[test$normalized$time == conditions[i],])+geom_line()+ggtitle(conditions[i])+xlim(-50,50)+ylim(0.003,0.018)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))+ylab("Frequency")
    plots[[i]] <- p1  # add each plot into plot list
}
multiplot(plotlist = plots, cols = 2)
```

```{r,echo = FALSE}

hmc_48_plus = read.delim("../data/Data/160620/He-Lu-6_48ng-S3_L001_R1_001.adaptor_removed.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

hmc_24_plus = read.delim("../data/Data/160620/He-Lu-7_24ng-S1_L001_R1_001.adaptor_removed.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

hmc_12_plus = read.delim("../data/Data/160620/He-Lu-8_12ng-S2_L001_R1_001.adaptor_removed.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

hmc_6_plus = read.delim("../data/Data/160620/He-Lu-9_6ng-S4_L001_R1_001.adaptor_removed.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

hmc_2.4_plus = read.delim("../data/Data/160620/He-Lu-10_2.4ng-S5_L001_R1_001.adaptor_removed.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

hmc_plus = cbind(hmc_48_plus,hmc_24_plus,hmc_12_plus,hmc_6_plus,hmc_2.4_plus)
colnames(hmc_plus) = c("48ng","24ng","12ng","6ng","2.4ng")
hmc_plus = data.frame(hmc_plus, position = seq(-100,100,1))
```

\newpage
* hmc plus strands minus hydroxymethyl sites, unnormalized

```{r,echo=FALSE,fig.width=5,fig.height=5}
test = orgnize_table(hmc_plus)
ggplot(aes(x = position, y = count, color = time), data = test$original)+geom_line()+scale_color_discrete(guide = guide_legend(title = "Condition"))+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black",hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"),legend.position = c(.9, .8))+xlab("Distance")+ylab("Count")
```


```{r,echo=FALSE}
plots <- list()  # new empty list
for (i in 1:5) {
    p1 = ggplot(aes(x = position, y = count), data = test$original[test$original$time == conditions[i],])+geom_line()+ggtitle(conditions[i])+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))
    plots[[i]] <- p1  # add each plot into plot list
}
multiplot(plotlist = plots, cols = 2)
```

\newpage
* hmc plus strands minus hydroxymethyl sites, normalized

```{r,echo=FALSE,fig.width=5, fig.height=5}
ggplot(aes(x = position, y = count, color = time), data = test$normalized)+geom_line()+scale_color_discrete(guide = guide_legend(title = "Condition"))+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black",hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"),legend.position = c(.9, .8))+xlab("Distance")+ylab("Frequency")
```

```{r,echo=FALSE}
plots <- list()  # new empty list
for (i in 1:5) {
    p1 = ggplot(aes(x = position, y = count), data = test$normalized[test$normalized$time == conditions[i],])+geom_line()+ggtitle(conditions[i])+xlim(-50,50)+ylim(0.003,0.018)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))+ylab("Frequency")
    plots[[i]] <- p1  # add each plot into plot list
}
multiplot(plotlist = plots, cols = 2)
```

```{r,echo=FALSE}
input_48_plus = read.delim("../data/Data/160620/He-Lu-1_input-48ng-S1_L004_R1_001.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

input_24_plus = read.delim("../data/Data/160620/He-Lu-2_input-24ng-S2_L004_R1_001.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

input_12_plus = read.delim("../data/Data/160620/He-Lu-3_input-12ng-S3_L004_R1_001.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

input_6_plus = read.delim("../data/Data/160620/He-Lu-4_input-6ng-S4_L004_R1_001.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

input_2.4_plus = read.delim("../data/Data/160620/He-Lu-5_input-2.4ng-S5_L004_R1_001.bam.plus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

input_plus = cbind(input_48_plus, input_24_plus, input_12_plus, input_6_plus, input_2.4_plus)
colnames(input_plus) = c("48ng","24ng","12ng","6ng","2.4ng")
input_plus = data.frame(input_plus, position = seq(-100,100,1))

```

\newpage
* input plus strands minus hydroxymethyl sites, unnormalized

```{r,echo=FALSE,fig.width=5,fig.height=5}
test = orgnize_table(input_plus)
ggplot(aes(x = position, y = count, color = time), data = test$original)+geom_line()+scale_color_discrete(guide = guide_legend(title = "Condition"))+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black",hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"),legend.position = c(.9, .8))+xlab("Distance")+ylab("Count")
```


```{r,echo=FALSE}
plots <- list()  # new empty list
for (i in 1:5) {
    p1 = ggplot(aes(x = position, y = count), data = test$original[test$original$time == conditions[i],])+geom_line()+ggtitle(conditions[i])+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))
    plots[[i]] <- p1  # add each plot into plot list
}
multiplot(plotlist = plots, cols = 2)
```
\newpage
* input plus strands minus hydroxymethyl sites, normalized


```{r,echo=FALSE,fig.width=5, fig.height=5}
ggplot(aes(x = position, y = count, color = time), data = test$normalized)+geom_line()+scale_color_discrete(guide = guide_legend(title = "Condition"))+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black",hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"),legend.position = c(.9, .8))+xlab("Distance")+ylab("Frequency")
```

```{r,echo=FALSE}
plots <- list()  # new empty list
for (i in 1:5) {
    p1 = ggplot(aes(x = position, y = count), data = test$normalized[test$normalized$time == conditions[i],])+geom_line()+ggtitle(conditions[i])+xlim(-50,50)+ylim(0.000,0.018)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))+ylab("Frequency")
    plots[[i]] <- p1  # add each plot into plot list
}
multiplot(plotlist = plots, cols = 2)
```


\newpage
### minus reads and plus sites

```{r,echo=FALSE}
#minus reads and minus sites
setwd("/media/yuwen/Elements/Tiao_Seq/analysis")
mc5_48_minus = read.delim("../data/Data/160620/He-Lu-6_5mC-jump-48ng-S6_L004_R1_001.adaptor_removed.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

mc5_24_minus = read.delim("../data/Data/160620/He-Lu-7_5mC-jump-24ng-S7_L004_R1_001.adaptor_removed.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)


mc5_12_minus = read.delim("../data/Data/160620/He-Lu-8_5mc-jump-12ng-S8_L004_R1_001.adaptor_removed.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)


mc5_6_minus = read.delim("../data/Data/160620/He-Lu-9_5mc-jump-6ng-S9_L004_R1_001.adaptor_removed.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

mc5_2.4_minus = read.delim("../data/Data/160620/He-Lu-10_5mc-jump-2.4ng-S10_L004_R1_001.adaptor_removed.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)


mc5_minus = cbind(mc5_48_minus,mc5_24_minus,mc5_12_minus,mc5_6_minus,mc5_2.4_minus)
colnames(mc5_minus) = c("48ng","24ng","12ng","6ng","2.4ng")
mc5_minus = data.frame(mc5_minus, position = seq(-100,100,1))
```

```{r,echo=FALSE}
knitr::opts_chunk$set(fig.width = 10,fig.height = 5)
options(warn=-1)
```



* 5mc minus strands plus hydroxymethyl sites, unnormalized

```{r,echo=FALSE,fig.width=5,fig.height=5}
test = orgnize_table(mc5_minus)
ggplot(aes(x = position, y = count, color = time), data = test$original)+geom_line()+scale_color_discrete(guide = guide_legend(title = "Condition"))+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black",hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"),legend.position = c(.9, .8))+xlab("Distance")+ylab("Count")
```

```{r,echo=FALSE}
plots <- list()  # new empty list
for (i in 1:5) {
    p1 = ggplot(aes(x = position, y = count), data = test$original[test$original$time == conditions[i],])+geom_line()+ggtitle(conditions[i])+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))
    plots[[i]] <- p1  # add each plot into plot list
}
multiplot(plotlist = plots, cols = 2)
```

\newpage
* 5mc minus strands plus hydroxymethyl sites, normalized

```{r,echo=FALSE,fig.width=5,fig.height=5}
ggplot(aes(x = position, y = count, color = time), data = test$normalized)+geom_line()+scale_color_discrete(guide = guide_legend(title = "Condition"))+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black",hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"),legend.position = c(.9, .8))+xlab("Distance")+ylab("Frequency")
```

```{r,echo=FALSE}
plots <- list()  # new empty list
for (i in 1:5) {
    p1 = ggplot(aes(x = position, y = count), data = test$normalized[test$normalized$time == conditions[i],])+geom_line()+ggtitle(conditions[i])+xlim(-50,50)+ylim(0.003,0.018)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))+ylab("Frequency")
    plots[[i]] <- p1  # add each plot into plot list
}
multiplot(plotlist = plots, cols = 2)
```

```{r,echo = FALSE}

hmc_48_minus = read.delim("../data/Data/160620/He-Lu-6_48ng-S3_L001_R1_001.adaptor_removed.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

hmc_24_minus = read.delim("../data/Data/160620/He-Lu-7_24ng-S1_L001_R1_001.adaptor_removed.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

hmc_12_minus = read.delim("../data/Data/160620/He-Lu-8_12ng-S2_L001_R1_001.adaptor_removed.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

hmc_6_minus = read.delim("../data/Data/160620/He-Lu-9_6ng-S4_L001_R1_001.adaptor_removed.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

hmc_2.4_minus = read.delim("../data/Data/160620/He-Lu-10_2.4ng-S5_L001_R1_001.adaptor_removed.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

hmc_minus = cbind(hmc_48_minus,hmc_24_minus,hmc_12_minus,hmc_6_minus,hmc_2.4_minus)
colnames(hmc_minus) = c("48ng","24ng","12ng","6ng","2.4ng")
hmc_minus = data.frame(hmc_minus, position = seq(-100,100,1))
```

\newpage
* hmc minus strands plus hydroxymethyl sites, unnormalized

```{r,echo=FALSE,fig.width=5,fig.height=5}
test = orgnize_table(hmc_minus)
ggplot(aes(x = position, y = count, color = time), data = test$original)+geom_line()+scale_color_discrete(guide = guide_legend(title = "Condition"))+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black",hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"),legend.position = c(.9, .8))+xlab("Distance")+ylab("Count")
```

```{r,echo=FALSE}
plots <- list()  # new empty list
for (i in 1:5) {
    p1 = ggplot(aes(x = position, y = count), data = test$original[test$original$time == conditions[i],])+geom_line()+ggtitle(conditions[i])+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))
    plots[[i]] <- p1  # add each plot into plot list
}
multiplot(plotlist = plots, cols = 2)
```

\newpage
* hmc minus strands plus hydroxymethyl sites, normalized

```{r,echo=FALSE,fig.width=5,fig.height=5}
ggplot(aes(x = position, y = count, color = time), data = test$normalized)+geom_line()+scale_color_discrete(guide = guide_legend(title = "Condition"))+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black",hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"),legend.position = c(.9, .8))+xlab("Distance")+ylab("Frequency")
```

```{r,echo=FALSE}
plots <- list()  # new empty list
for (i in 1:5) {
    p1 = ggplot(aes(x = position, y = count), data = test$normalized[test$normalized$time == conditions[i],])+geom_line()+ggtitle(conditions[i])+xlim(-50,50)+ylim(0.003,0.018)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))+ylab("Frequency")
    plots[[i]] <- p1  # add each plot into plot list
}
multiplot(plotlist = plots, cols = 2)
```

```{r,echo=FALSE}
input_48_minus = read.delim("../data/Data/160620/He-Lu-1_input-48ng-S1_L004_R1_001.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

input_24_minus = read.delim("../data/Data/160620/He-Lu-2_input-24ng-S2_L004_R1_001.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

input_12_minus = read.delim("../data/Data/160620/He-Lu-3_input-12ng-S3_L004_R1_001.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

input_6_minus = read.delim("../data/Data/160620/He-Lu-4_input-6ng-S4_L004_R1_001.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

input_2.4_minus = read.delim("../data/Data/160620/He-Lu-5_input-2.4ng-S5_L004_R1_001.bam.minus_sites_with_1_coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

input_minus = cbind(input_48_minus, input_24_minus, input_12_minus, input_6_minus, input_2.4_minus)
colnames(input_minus) = c("48ng","24ng","12ng","6ng","2.4ng")
input_minus = data.frame(input_minus, position = seq(-100,100,1))

```

\newpage
* input minus strands plus hydroxymethyl sites, unnormalized


```{r,echo=FALSE,fig.width=5,fig.height=5}
test = orgnize_table(input_minus)
ggplot(aes(x = position, y = count, color = time), data = test$original)+geom_line()+scale_color_discrete(guide = guide_legend(title = "Condition"))+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black",hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"),legend.position = c(.9, .8))+xlab("Distance")+ylab("Count")
```

```{r,echo=FALSE}
plots <- list()  # new empty list
for (i in 1:5) {
    p1 = ggplot(aes(x = position, y = count), data = test$original[test$original$time == conditions[i],])+geom_line()+ggtitle(conditions[i])+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))
    plots[[i]] <- p1  # add each plot into plot list
}
multiplot(plotlist = plots, cols = 2)
```

\newpage
* input minus strands plus hydroxymethyl sites, normalized

```{r,echo=FALSE,fig.width=5,fig.height=5}
ggplot(aes(x = position, y = count, color = time), data = test$normalized)+geom_line()+scale_color_discrete(guide = guide_legend(title = "Condition"))+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black",hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"),legend.position = c(.9, .8))+xlab("Distance")+ylab("Frequency")
```

```{r,echo=FALSE}
plots <- list()  # new empty list
for (i in 1:5) {
    p1 = ggplot(aes(x = position, y = count), data = test$normalized[test$normalized$time == conditions[i],])+geom_line()+ggtitle(conditions[i])+xlim(-50,50)+ylim(0.000,0.018)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))+ylab("Frequency")
    plots[[i]] <- p1  # add each plot into plot list
}
multiplot(plotlist = plots, cols = 2)
```