---
title: "plot Jump-seq reads distribution"
output: html_document
---

Alignment paramters:
Skip the 1st base
Doesn't allow any mismatch
collapse PCR duplicates.


### plus reads and minus sites
```{r,echo=FALSE}
library(ggplot2)
library(Rmisc)
#function to return a data.frame that could be used to draw plots
orgnize_table <-function(data){
  #data is a table in which each column represents under one condition the counts of 5' end of reads across -100 to +100 of hydroxymethylation sites
  data2 = as.data.frame(cbind(apply(data[,1:5],2,function(x){x/sum(x)}),position = data$position)) # normalize for each column of the first 5
  data2 = reshape(data2, varying = list(1:5),v.names = "count",idvar = "position",times = c("48ng","24ng","12ng","6ng","2.4ng"),direction = "long")
  data1 = reshape(data, varying = list(1:5),v.names = "count",idvar = "position",times = c("48ng","24ng","12ng","6ng","2.4ng"),direction = "long")
  data1$time = factor(data1$time, levels = c("48ng","24ng","12ng","6ng","2.4ng"))
  data2$time = factor(data2$time, levels = c("48ng","24ng","12ng","6ng","2.4ng"))
  list(original = data1, normalized = data2)
}

### a vector containing different conditions
conditions = c("48ng","24ng","12ng","6ng","2.4ng")
```



```{r,echo=FALSE}
#plus reads and minus sites
setwd("/media/yuwen/Elements/Tiao_Seq/analysis")
spike_in_2_site = read.delim("../data/Data/20161116/20161115_5hmC_Jump_Seq_2hmC_spike_in.161116_5hmc_2sites.umi_encoded_adaptor_removed_no_mismatch.sorted.dedup.bam.minus.sorted.5prime.bed.coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

spike_in_2_site_data = data.frame(coverage = spike_in_2_site[,1], position = seq(-97,102))

spike_in_1_site = read.delim("../data/Data/20161116/20161115_5hmC_Jump_Seq_positive_control_spike_in.160811_5hmc_spikein.umi_encoded_adaptor_removed_no_mismatch.sorted.dedup.bam.minus.sorted.5prime.bed.coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

spike_in_1_site_data = data.frame(coverage = spike_in_1_site[,1], position = seq(-97,102))

spike_in_0_site = read.delim("../data/Data/20161116/20161115_5hmC_Jump_Seq_negative_control_spike_in.160811_5hmc_spikein.umi_encoded_adaptor_removed_no_mismatch.sorted.dedup.bam.minus.sorted.5prime.bed.coverage.ave.perbase", header = FALSE, sep = "\t", stringsAsFactors = FALSE)

spike_in_0_site_data = data.frame(coverage = spike_in_0_site[,1], position = seq(-97,102))


```

```{r,echo=FALSE}
library(ggplot2)
knitr::opts_chunk$set(fig.width = 6,fig.height = 4)
options(warn=-1)
```


```{r,echo=FALSE}
ggplot(aes(x = position, y = coverage), data = spike_in_2_site_data) + geom_line()+ggtitle("2-site")+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))+geom_vline(xintercept = c(-8,0), colour = "red", linetype = "longdash")
```

```{r,echo=FALSE}
ggplot(aes(x = position, y = coverage), data = spike_in_1_site_data) + geom_line()+ggtitle("1-site")+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))+geom_vline(xintercept = 0, colour = "red", linetype = "longdash")
```

```{r,echo=FALSE}
ggplot(aes(x = position, y = coverage), data = spike_in_0_site_data) + geom_line()+ggtitle("0-site")+xlim(-50,50)+theme_classic()+theme(axis.text.x = element_text(size =10, color = "black", hjust = 1, vjust = 1), axis.text.y = element_text(size =10, color = "black"))
```

