---
title: "Single cell paired end reads mapping pipeline"
output: html_document
---

***

* [pipeline using](../code/161213_bioinformatics_preprocessing_using_UMI_paired_end_demultiplexing.sos) `../code/161213_bioinformatics_preprocessing_using_UMI_paired_end_demultiplexing.sos`

* Data: `../data/Data/20161130/`

* Major steps

```
1. Encode the first 8 bases as UMI.
2. Only retain reads whose 1st read has Jump-barcode.
3. Trim reads that have adaptor sequences at the 3'ends, for both reads of each pair.
4. Remove a pair of reads if any one of them is empty.
5. Demultiplexing using cell-specific indexes.
6. Paired end mapping without mismatch (no mismatch in the first 35 base, maximal insert size is 250, not allow multiple alignment).
7. collapsing PCR duplicates using UMI information (if not mappable in Step 6, reads won't be kept after Step 7). 
```

* Needs to be updated in the pipeline

```
1. count reads number for each fastq and store the number for each fastq file.
```
