<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/yeti.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>




<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 45px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 50px;
  margin-top: -50px;
}

.section h2 {
  padding-top: 50px;
  margin-top: -50px;
}
.section h3 {
  padding-top: 50px;
  margin-top: -50px;
}
.section h4 {
  padding-top: 50px;
  margin-top: -50px;
}
.section h5 {
  padding-top: 50px;
  margin-top: -50px;
}
.section h6 {
  padding-top: 50px;
  margin-top: -50px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Collaboration J Project</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="meeting.html">Meeting</a>
</li>
<li>
  <a href="Note.html">Note</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<h1 style="color:grey;font-family:arial; position:relative; top:15px;text-align:center;">
Collaboration J project<img src="website_materials/research_website_logo.png"  style="float:right;width:200px;height:150px;">
</h1>
<p><br></p>
<hr />
<div id="available-data-set" class="section level3">
<h3>0. <strong>Available data set</strong></h3>
<ul>
<li>Project data list shared in Google drive <a href="https://drive.google.com/open?id=1WesFCQTrFplnXRKCHmhqtm1529DHc5Sc2tX7GkRa_-U">Click…</a>.</li>
</ul>
<p><br></p>
</div>
<div id="spike-in-distribution" class="section level3">
<h3>1. <strong>spike-in distribution</strong></h3>
<ul>
<li><p>161116 1-site,2-site,and negative control, stringent mapping, PCR duplicates removed <a href="./161120_161116_spike_in_data_plot_reads_distribution.html">[More…]</a>.</p></li>
<li><p>160917_160811 1-site spike-in, mapping allowing for mismatches, PCR duplicates removed <a href="./160912_plot_reads_distribution_UMI_dedupped.html">[More…]</a>.</p></li>
<li><p>170120_first, mapping not allowing for mismatches, PCR duplicates removed <a href="./170204_170120_first_batch_spike_in_plot_reads_distribution.html">[More…]</a>.</p></li>
</ul>
<p><br></p>
</div>
<div id="real-data-distribution" class="section level3">
<h3>2. <strong>Real data distribution</strong></h3>
<ul>
<li><p>160912_160620, compared to Tab-Seq all peaks, mapping allowing mismatches, PCR duplicates removed <a href="./160912_plot_reads_distribution_UMI_dedupped.html">[More…]</a>.</p></li>
<li><p>161120_160620, compared to Tab-Seq FDR 0.484 peaks, mapping allowing mismatches, PCR duplicates removed <a href="./161120_160620_data_plot_reads_distribution.html">[More…]</a>.</p></li>
<li><p>170120_first data, compared to Tab-Seq FDR 0.484 peaks, mapping not allowing mismatches, PCR duplicates removed <a href="./170204_170120_first_batch_data_plot_reads_distribution.html">[More…]</a>.</p></li>
</ul>
<p><br></p>
</div>
<div id="shengtong-peak-overlap-analysis" class="section level3">
<h3>3. <strong>Shengtong peak overlap analysis</strong></h3>
<ul>
<li>drop_box_share made by Shengtong Han <a href="./JumpSeq_dropbox_shengtong.pdf">[More…]</a>.</li>
</ul>
<p><br></p>
</div>
<div id="paired-end-sequencing-analysis" class="section level3">
<h3>4. <strong>Paired end sequencing analysis</strong></h3>
<ul>
<li><p>161218 analysis pipeline <a href="./161218_paired_end_reads_mapping_pipeline.html">[More…]</a>.</p></li>
<li><p>161218 pipeline, 20161130 5hmc data, processed reads number and mappable reads number <a href="./161218_161130_paired_end_reads_statistics.html">[More…]</a>.</p></li>
<li><p>161218 pipeilne, 20161130 5mc data, processed reads number and mapped reads number <a href="./161222_161130_5mc_paired_end_reads_statistics.html">[More…]</a>.</p></li>
<li><p>161218 pipeline, 20170110 5mc data, processed reads number and mapped reads number <a href="./170115_170110_paired_end_reads_statistics.html">[More…]</a>.</p></li>
</ul>
<p><br></p>
</div>
<div id="distinct-reads-saturation-analysis" class="section level3">
<h3>5. <strong>distinct reads Saturation analysis</strong></h3>
<ul>
<li>161226 analysis for all previous 5hmc 48 ng data and one 2.4 ng data <a href="./161226_previous_5hmc_distinct_reads_saturation_analysis.html">[More…]</a></li>
</ul>
<p><br></p>
</div>
<div id="investigation-on-reads-lost-at-each-preprocessing-step" class="section level3">
<h3>6. <strong>Investigation on reads lost at each preprocessing step</strong></h3>
<ul>
<li><p>Investiage reads lost at each step for all previous up to 160811 48ng 5hmc data <a href="./170207_previous_5hmc_samples_reads_lost_at_each_preprocessing_step.html">[More…]</a>.</p></li>
<li><p>Explore why there is a low mappability rate for reads with a Jump-Seq barcode. Use 160402 5hmc data (He-lu-6_S6_L006_R1_001). Randomly picked 10 reads, 6 reads have multiple alignments, 6 reads would have been mapped if allowing for 2 or 3 mismatches, 3 reads are not mapped even if allowing for 3 mismatches <a href="./170210_investigate_low_mappability_issue.html">[More…]</a>.</p></li>
<li><p>170120_second, cell free DNA Jump-Seq (323T_hmC_jump_cfDNA) reads statistics at each step <a href="./170210_20170120_second_human_cf_5hmc_samples_reads_lost_at_each_preprocessing_step.html">[More…]</a>.</p></li>
</ul>
<p><br></p>
</div>
<div id="peaks-compariosn" class="section level3">
<h3>7. Peaks compariosn</h3>
<ul>
<li>compare gDNA hmc seal peaks from tumor samples to from adjacent tissues. Also compare cell-free DNA hmc seal sites with these sites <a href="./170210_20170120_second_human_cf_5hmc_samples_peak_comparison.html">[More…]</a>.</li>
</ul>
<hr />
<p><strong>Last updated:</strong> 2017-03-03 21:12:10</p>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
